changelog.d/1556

```diff
@@ -0,0 +1,18 @@
+synopsis: Display capture hints in router layout
+prs: #1556
+
+description: {
+
+This PR introduces a CaptureHint type, which is passed as an extra argument to the CaptureRouter and CaptureAllRouter constructors for the Router' type.
```

Comment:
I think this changelog entry is a bit too technical from the start — it does not inform bystanders very well on what the end result is.

Maybe starting with an before / after example would be a good idea ?